lib/cloud_controller/serializer.rb

```diff
@@ -7,7 +7,13 @@ def serializes_via_json(accessor_method_name)
         string = send("#{accessor_method_name}_without_serialization")
         return if string.blank?
 
-        MultiJson.load string
+        begin
+          MultiJson.load string
```

Comment:
There seem to be other places where JSON data is loaded in a similar way, e.g. https://github.com/cloudfoundry/cloud_controller_ng/blob/e719d017ea4625397a97c2c14352ebdee66febe9/app/models/runtime/environment_variable_group.rb#L35

I think we should have a similar exception handling there as well.

Reply:
Strangely enough, I was unable to reproduce this error through the api, like how I was for applications. Perhaps this is only used on the backend?

Reply:
@philippthun lmk if you have any further questions, or concerns